storage/google/cloud/storage/bucket.py

```diff
@@ -280,6 +280,12 @@ class IAMConfiguration(dict):
     :type bucket: :class:`Bucket`
     :params bucket: Bucket for which this instance is the policy.
 
+    :type uniform_bucket_level_access_enabled: bool
+    :params bucket_policy_only_enabled: (optional) whether the IAM-only policy is enabled for the bucket.
+
+    :type uniform_bucket_level_locked_time: :class:`datetime.datetime`
+    :params uniform_bucket_level_locked_time: (optional) When the bucket's IAM-only policy was ehabled.  This value should normally only be set by the back-end API.
+
     :type bucket_policy_only_enabled: bool
     :params bucket_policy_only_enabled: (optional) whether the IAM-only policy is enabled for the bucket.
```

Comment:
Should `bucket_policy_only_enabled` and `bucket_policy_only_locked_time` be documented as deprecated?